system/metrics/sinks/elastic/elasticSink.cpp

```diff
@@ -193,3 +311,17 @@ void ElasticMetricSink::collectingHasStopped()
 
 }
 
+
+bool ElasticMetricSink::validateIndex()
+{
+    std::string endpoint;
+    endpoint.append("/").append(indexName.str());
+    auto res = pClient->Get(endpoint.c_str(), elasticHeaders);
```

Comment:
what is the connect timeout value for this connection attempt?

Reply:
is pClient only used to extract index info? if we intent to use it for other functionality, the above timeout values might not be reasonable.